frontend/amundsen_application/static/js/config/config-utils.ts

```diff
@@ -474,6 +474,13 @@ export function isTableQualityCheckEnabled() {
   return AppConfig.tableQualityChecks.isEnabled;
 }
 
+/**
+ * Returns whether dangerous html should be allowed in programmatic descriptions
+ */
+export function isProgrammaticDescAllowDangerousHtml(): boolean {
```

Comment:
this is a mouthful of name. It is tricky, but I think 'hasHTMLDescription' would work better. Same for the config object field.